src/testSvc/conftest.py

```diff
@@ -31,3 +31,13 @@ def generate_workflow_target_event(setup_database_connection):
     yield event_data
     mongo_session = setup_database_connection
     mongo_session.delete_one({'Event_name': target_event_name})
+
+
+@pytest.fixture(scope='class')
+def add_player(self, generate_workflow_target_event, setup_database_connection):
+        event_data = generate_workflow_target_event
+        mongo_session = setup_database_connection
+        target_endpoint = f'{DEFAULT_TARGET_ENV}/event-manager/add-player/{event_data["Event_id"]}'
+        player_name = ''.join('someone')
+        player_commander = ''.join('sometwo')

```

Comment:
As I said, he have to use fixture factory here.
If we implement this case like that, we get many players with the same name and commander.
See there, lines 27-28 https://github.com/CustomCommanderReporter/commanderReporter/blob/main/src/testSvc/coreWorkflow/BaseWorkflow.py
You can make factory which generates name and commander strings and make request to add player in test.